src/eloqua.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+    if (!CerosSDK) {
+        var sdkScript = document.createElement('script');
+        sdkScript.type = "text/javascript";
+        sdkScript.async = true;
+        sdkScript.onload = activateEloquaTracking;
+
+        // Are we embedded in an iframe?
```

Comment:
Wouldn't embedded experiences still need the standalone SDK?
The embedded one is loaded and used by the parent page